signature/cose/cose.go

```diff
@@ -1,25 +1,155 @@
 package cose
 
 import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"errors"
 	"fmt"
+	"io"
+	"time"
 
 	"github.com/notaryproject/notation-core-go/signature"
 	"github.com/notaryproject/notation-core-go/signature/internal/base"
+	nx509 "github.com/notaryproject/notation-core-go/x509"
+	cosepkg "github.com/veraison/go-cose"
```

Comment:
Any reason to alias `go-cose` to `cosepkg` instead of `cose`?

Note: We can reference `cose` inside the package named `cose`.